### DISCUSSION TITLE
Bug with payments on the 30th in FEB

### DISCUSSION THREAD

**Discussion by shaunVorster** [ellite/Wallos#565] [Q&A]
I think there may be an issue with payments on the 29/30th of the month when we hit feb.

I see my one payment was moved to the 2nd of the the month following form march onwards.
Not sure if that is what was intended.

**Comment by ellite** [comment]
Hi, Please check the discussion here:

https://github.com/ellite/Wallos/issues/551

It's not intended but I don't have a solution yet.

**Reply by shaunVorster** [reply]
Thanks, will do
